packages/agreement-email-sender/src/services/agreementEmailSenderService.ts

```diff
@@ -96,34 +73,142 @@ async function getActivationMailFromAgreement(
     throw descriptorNotFound(agreement.eserviceId, agreement.descriptorId);
   }
 
-  return {
+  const mail = {
     subject: `Richiesta di fruizione ${agreement.id} attiva`,
     to: [producerEmail, consumerEmail],
     body: templateService.compileHtml(htmlTemplate, {
-      activationDate: formattedActivationDate,
+      activationDate,
       agreementId: agreement.id,
       eserviceName: eservice.name,
       eserviceVersion: descriptor.version,
       producerName: producer.name,
       consumerName: consumer.name,
     }),
   };
+
+  logger.info(`Sending email for agreement ${agreement.id} activation`);
+  await emailManager.send(
+    agreementEmailSender,
+    mail.to,
+    mail.subject,
+    mail.body
+  );
+  logger.info(`Email sent for agreement ${agreement.id} activation`);
 }
 
-// eslint-disable-next-line max-params
-export async function sendAgreementEmail(
-  agreement: AgreementV2,
+export async function sendAgreementSubmissionMail(
+  agreementV2: AgreementV2,
   readModelService: ReadModelService,
   emailManager: EmailManager,
   logger: Logger,
-  { agreementEmailSender } = agreementEmailSenderConfig()
+  { awsSesAgreementEmailSender } = agreementEmailSenderConfig()
 ): Promise<void> {
-  const { to, subject, body } = await getActivationMailFromAgreement(
+  const agreement = fromAgreementV2(agreementV2);
+  const templateService = buildHTMLTemplateService();
+  const htmlTemplate = await retrieveHTMLTemplate("submission-mail");
+
+  const { eservice, producer, consumer } = await retrieveAgreementComponents(
     agreement,
     readModelService
   );
 
-  logger.info(`Sending email for agreement ${agreement.id} activation`);
-  await emailManager.send(agreementEmailSender, to, subject, body);
-  logger.info(`Email sent for agreement ${agreement.id} activation`);
+  const submissionDate = getFormattedAgreementStampDate(
+    agreement,
+    "submission"
+  );
+
+  const producerEmail = [...producer.mails]
+    .filter((m) => m.kind === "CONTACT_EMAIL")
+    .sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())[0];
+
+  if (!producerEmail) {
+    logger.warn(
+      `Producer email not found for agreement ${agreement.id}, skipping email`
+    );
+    return;
+  }
+
+  const mail = {
+    from: awsSesAgreementEmailSender,
+    subject: `Nuova richiesta di fruizione per ${eservice.name} ricevuta`,
+    to: [producerEmail.address],
+    body: templateService.compileHtml(htmlTemplate, {
+      interopFeUrl: `https://selfcare.interop.pagopa.it/ui/it/erogazione/richieste/${agreement.id}`,
```

Comment:
Should we set the url in the env variables? This could be helpful to change it based on the environment, ex:

`DEV` -> `selfcare.dev.interop.pagopa.it`
`UAT` -> `selfcare.uat.interop.pagopa.it`

Reply:
yes, this should be an env var, thanks!